packages/nodes-base/nodes/LocalFileTrigger/LocalFileTrigger.node.ts

```diff
@@ -208,19 +208,20 @@ export class LocalFileTrigger implements INodeType {
 						type: 'options',
 						options: [
 							{
-								name: 'Regex',
-								value: 'regex',
-								description: 'Use regex patterns (e.g., **/*.txt)',
+								name: 'Match',
+								value: 'match',
```

Comment:
I would keep value the same here, or existing nodes that use this will no longer work